docs/source/support/release-notes.rst

```diff
@@ -31,6 +31,12 @@ DAML Studio
 
 - The IDE now executes tasks in parallel.
 
+Sandbox
+~~~~~~~~
+
+- Introduced a new API for package management.
```

Comment:
Does this deliberately appear twice?

Reply:
No, this is an error introduced after rebasing, fixed now.